src/pages/OfferNew/Checkout/Sign.tsx

```diff
@@ -51,6 +43,14 @@ const ButtonWrapper = styled('div')`
   justify-content: center;
 `
 
+const SpinnerWrapper = styled(motion.div)`
+  display: inline-block;
+  padding-left: 0.5em;
+  margin-top: -1px;
+  vertical-align: text-top;
+  overflow: hidden;
+`
```

Comment:
Export from the spinner util file maybe?

Reply:
It's a nice catch. However, I built the spinner to be context-independent and this, to me, is the context. They should imo be separated from the spinner itself. Idk if that makes sense, or?